packages/suite/src/components/suite/WordInput/index.tsx

```diff
@@ -45,9 +45,21 @@ const SelectWrapper = styled.div`
 type Option = { label: string; value: string };
 
 const MenuList = (props: MenuListComponentProps<Option, boolean>) => {
+    const listRef = useRef<List>(null);
+
     const children = React.Children.toArray(props.children);
+
+    useEffect(() => {
+        if (listRef.current) {
+            const getFocusIndex = (children: any) =>
+                children.findIndex((child: any) => child.props.isFocused === true);
+            const currentIndex = getFocusIndex(children);
+            listRef.current.scrollToItem(currentIndex);
+        }
+    }, [children]);
```

Comment:
I would do it like this, but yea if some TS guru master checks it, it would be nice. 
```suggestion
    const children = React.Children.toArray(props.children) as React.ReactElement[];

    useEffect(() => {
        if (listRef.current) {
            const currentIndex = children.findIndex(child => child?.props.isFocused === true);
            listRef.current.scrollToItem(currentIndex);
        }
    }, [children]);
```

Reply:
I see 3 UI problems there.
https://user-images.githubusercontent.com/33235762/173071807-b3dfa415-1f4c-4b6c-b7e8-fecf410f9d59.mov

1. There is no spacing on top and right. (bug on production)
2. If I put the cursor on a word and I use a keyboard, there is still a hover on the word where the cursor is placed.
3. If I hold the arrow up/down, it looks like it does nothing but when I release the key it jumps to some word. (It is okay if I insert 1 letter)

Reply:
I wanted to avoid type casting, but this is definitely better. f99eb9bff7216416f069322f34e684b2b7e7dbf4

Ad 1) I fixed that and also added box-shadow to match other Selects. 61117a4b08fcdea5a228aad4cb69c082e91ea86c
Ad 2) 8bd8eb165203c0a7729ca07092477f1fde88122d
Ad 3) This looks like a performance issue and I cannot reproduce it on my M1. Probably not a big deal for users, so I would suggest keeping it as is.

Reply:
2 and 3) is ok now

Reply:
1) ok